pkg/controller/model/grafanaDeployment.go

```diff
@@ -116,8 +116,8 @@ func getReplicas(cr *v1alpha1.Grafana) *int32 {
 }
 
 func getRollingUpdateStrategy() *v1.RollingUpdateDeployment {
-	var maxUnaval intstr.IntOrString = intstr.FromInt(25)
-	var maxSurge intstr.IntOrString = intstr.FromInt(25)
+	var maxUnaval intstr.IntOrString = intstr.FromString("25%")
+	var maxSurge intstr.IntOrString = intstr.FromString("25%")
```

Comment:
@pb82  when you can, have a quick look at this, I think it should fix the 503 between grafana restarts